spec/models/personal_details_spec.rb

```diff
@@ -50,14 +50,6 @@
         end
       end
 
-      context "when there's an existing profile" do
-        before { create(:jobseeker_profile, jobseeker:) }
-
-        it "does not set completed steps" do
-          expect(personal_details.completed_steps).to be_empty
-        end
-      end
```

Comment:
This test setup made nonsense with the `Jobseeker has_one jobseeker_profile` association. The introduction of a unique index made this raise an exception. Removing this test as is bases in a non-valid test setup.